Assets/MixedRealityToolkit/_Core/Devices/BaseController.cs

```diff
@@ -90,16 +85,16 @@ public void SetupConfiguration(Type controllerType)
         /// Load the Interaction mappings for this controller from the configured Controller Mapping profile
         /// </summary>
         /// <param name="mappings">Configured mappings from a controller mapping profile</param>
-        protected void AssignControllerMappings(MixedRealityInteractionMapping[] mappings)
+        public void AssignControllerMappings(MixedRealityInteractionMapping[] mappings)
```

Comment:
Do we also need/want AddControllerMapping() and RemoveControllerMapping()?

Reply:
Yes, if someone decides they want to use a new controller mapping at runtime.

Reply:
I think we should ***add a task for this later***.  The mappings are meant to be static for the MVP.  it's a very limited use case (in any VR solution at the moment) where you would want to dynamically change mappings on the fly.

Reply:
I remember this being a selling point with @Railboy